spec/spec.md

```diff
@@ -307,10 +308,10 @@ Design Principle #5 ([Cryptographic Verifiability](https://trustoverip.org/perma
   1. **[[xref: toip, Externally-verified identifiers]] (XVIDs)** are generated by an interaction between the VID controller's [[xref: toip, agent]] that has access to the [[xref: toip, digital wallet]] holding the cryptographic key pair and some type of external system or authority, such as a [[xref: toip, blockchain]], [[xref: toip, distributed hash table]], or [[xref: toip, certificate authority]] (CA) that is outside of the autonomous boundary of the [[xref: toip, VID controller]]. Verification of an XVID requires the [[xref: toip, verifier]] to interact with that [[xref: toip, supporting system]].
 
-Currently, the best-known form of VID is a [[xref: toip, decentralized identifier]] (DID) as defined by the [W3C Decentralized Identifiers (DID) V1.0 Specification] TODO-REFERENCE. In general, DIDs fulfill the requirement of Design Principle #4 ([Decentralization by Design and Default](https://trustoverip.org/permalink/Design-Principles-for-the-ToIP-Stack-V1.0-2022-11-17.pdf)). The W3C DID specification defines a generic syntax for DIDs and a standard data model for a [[xref: toip, DID document]]—the artifact obtained through [[xref: toip, DID resolution]] that contains the [[xref: toip, cryptographic keys]] and [[xref: toip, service endpoints]] bound to the DID. The syntax for a specific type of DID and the process for creating, reading, updating, or deactivating the associated DID document is defined by a [[xref: toip, DID method]]. As of mid-2024, there are almost 200 DID methods registered in the W3C DID Spec Registry, all of which can be classified as either a SCID or an XVID.
+Currently, the best-known form of VID is a [[xref: toip, decentralized identifier]] (DID) as defined by the [W3C Decentralized Identifiers (DID) V1.0 Specification] TODO-REFERENCE. In general, DIDs fulfill the requirement of Design Principle #4 ([Decentralization by Design and Default](https://trustoverip.org/permalink/Design-Principles-for-the-ToIP-Stack-V1.0-2022-11-17.pdf)). The W3C DID specification defines a generic syntax for DIDs and a standard data model for a [[xref: toip, DID document]]—the artifact obtained through [[xref: toip, DID resolution]] that contains the [[xref: toip, cryptographic keys]] and [[xref: toip, service endpoints]] bound to the DID. The syntax for a specific type of DID and the process for creating, reading, updating, or deactivating the associated DID document is defined by a [[xref: toip, DID method]]. As of mid-2024, there are almost 200 DID methods registered in the W3C DID Spec Registry, many, if not all, of which can be classified as either a SCID or an XVID.
```

Comment:
```suggestion
Currently, the best-known form of VID is a [[xref: toip, decentralized identifier]] (DID) as defined by the [W3C Decentralized Identifiers (DID) V1.0 Specification] `TODO-REFERENCE`. In general, DIDs fulfill the requirement of Design Principle #4 ([Decentralization by Design and Default](https://trustoverip.org/permalink/Design-Principles-for-the-ToIP-Stack-V1.0-2022-11-17.pdf)). The W3C DID specification defines a generic syntax for DIDs and a standard data model for a [[xref: toip, DID document]]—the artifact obtained through [[xref: toip, DID resolution]] that contains the [[xref: toip, cryptographic keys]] and [[xref: toip, service endpoints]] bound to the DID. The syntax for a specific type of DID and the process for creating, reading, updating, or deactivating the associated DID document is defined by a [[xref: toip, DID method]]. As of mid-2024, there are almost 200 DID methods registered in the W3C DID Spec Registry, many, if not all, of which can be classified as either a SCID or an XVID.
```
highlighting to make it easier to spot for later.

Reply:
There are a lot of these